Ginger/Ginger/ValueExpression/ValueExpressionEditorPage.xaml.cs

```diff
@@ -941,16 +942,26 @@ private void xTestButton_Click(object sender, RoutedEventArgs e)
         {
             if (mVE == null)
             {
-                if (mContext.Environment == null && mEnvs != null && mEnvs.Count > 0)
+                if (mContext.Environment == null && mEnvs != null && mEnvs.Count > 0 && WorkSpace.Instance.UserProfile.RecentEnvironment != Guid.Empty)
```

Comment:
Please keep old condition as is in case WorkSpace.Instance.UserProfile.RecentEnvironment is empty or null

Reply:
Implemented the requested change

Reply:
add else part to this if which will return first environment in case WorkSpace.Instance.UserProfile.RecentEnvironment is empty or null

Reply:
I have added the requested changes